### ISSUE TITLE
Makefile fatal error

### ISSUE THREAD

**Issue by TheCynosure** [nesbox/TIC-80#321]
I am getting the following message when trying to compile for linux64
```
lto1: fatal error: bytecode stream generated with LTO version 4.1 instead of the expected 5.2
```
I am currently running Lubuntu 17.04 and using gcc 6.3.

When I looked this up people were saying it might have been a problem because of using different compilers?

**Comment by nesbox** [comment]
yes, seems it's a problem with different compilers
try to build SDL lib and replace it with `lib/linux/libSDL2.a`

**Comment by TheCynosure** [comment]
Now I can get past it failling for SDL2 but it fails on the other libraries. What are these other libraries (libz, liblua, and libgif)? So that I can build them all myself.

**Comment by nesbox** [comment]
yes, you can find them here https://github.com/nesbox/3rd-party

**Comment by Rolzad73** [comment]
@TheCynosure I had problems initially building it, but I have made my way through the problems.

First of all, the TIC-80 makefile "clean" target was not working for me, so I changed it a bit, just because if I ever get errors, I like to start clean after making modifications.

FROM:
```
clean: $(TIC_O) $(TIC80_O)
     $(RM) $(TIC_O) $(TIC80_O)
```

TO:
```
clean: 
     $(RM) $(TIC_O) $(TIC80_O)
```

Then I did 2 things:
1. remove all *.a files from TIC-80/linux/libs
2. build lua manally from the 3rd party git repo as follows:
(start in 3rd-party/lua-5.3.1)
```
$ make linux
$ make local
```
3. then, from 3rd-party/lua-5.3.1/install/lib copy the liblua.a to the TIC-80/lib/linux dir
4. copy the contents of the 3rd-party/lua-5.3.1/install/include dir to TIC-80/include/lua
5. compile TIC-80
```
$ make linux64
```

Now, I think the revomal of SDL, libz and libgif references only worked because my Ubuntu 17 installation already had these dev packages installed from previous things that I had built.

I realize that right now this is a workaround to a proper build setup, but will at least get us going.
